go/libraries/doltcore/doltdb/indexeditor.go

```diff
@@ -0,0 +1,355 @@
+package doltdb
```

Comment:
Ignore this file. I got tired of dealing with import loops I just created a new indexeditor.go. Will def remove later